fedimint-cli/src/lib.rs

```diff
@@ -568,28 +575,38 @@ impl FedimintCli {
                 Ok(CliOutput::Raw(serde_json::to_value(metadata).unwrap()))
             }
             Command::Client(command) => {
-                let config = cli.load_config()?;
                 let client = cli
-                    .build_client_ng(&self.module_gens)
+                    .build_client_ng(&self.module_gens, &self.get_invite_code()?)
```

Comment:
Why would normal client commands require an invite code? The config should be in the DB at that point, if not we can still error.

Reply:
💯